.github/workflows/build_binaries.yml

```
@@ -35,10 +35,10 @@ jobs:
 
       - name: Allow git dependencies
         if: github.event_name != 'push'
-        run: >-
-          git config --global
-          url."https://oxfordionics:${{ secrets.GHA_TOKEN }}@github.com".insteadOf
-          ssh://git@github.com
+        run: |
+          git config --global url."https://oxfordionics:${{ inputs.GHA_TOKEN }}@github.com".insteadOf ssh://git@github.com
+          git config --global url."https://oxfordionics:${{ inputs.GHA_TOKEN }}@github.com/".insteadOf git@github.com:
```

```suggestion
          git config --global url."https://oxfordionics:${{ secrets.GHA_TOKEN }}@github.com".insteadOf ssh://git@github.com
          git config --global url."https://oxfordionics:${{ secrets.GHA_TOKEN }}@github.com/".insteadOf git@github.com:
```

`GHA_TOKEN` is declared as a secret at the top of this file. This might be different from poetry preamble.

---

sorry thanks